### DISCUSSION TITLE
Running more than one concurrent instances of the "artillery run-fargate..." command in a single AWS account

### DISCUSSION THREAD

**Discussion by manuelcarriernunes** [artilleryio/artillery#2224] [Artillery]
Hi, Artillery community. I work for a company where more than one team wants to run the "artillery run-fargate..." command to load test their microservice, but the teams want to share an AWS account. What must we do to ensure the AWS resources created by the teams when they run their load tests don't interfere with each other? How do teams isolate their executions?

**Comment by hassy** [comment]
Hi @manuelcarriernunes 👋 Related answer here: https://github.com/artilleryio/artillery/discussions/2223#discussioncomment-7304769

The main AWS resource quota that can cause contention is Fargate vCPU quota if running tests on Fargate: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/service-quotas.html#:~:text=Fargate%20On%2DDemand%20vCPU%20resource%20count Usually very quick to increase via a support request to AWS.

**Reply by manuelcarriernunes** [reply]
Awesome! Thanks for your quick response, @hassy.
